scionlab/admin.py

```diff
@@ -509,7 +511,23 @@ class VPNCreationForm(_CreateUpdateModelForm):
     Specialised ModelForm for VPN creation which will initialise key and cert
     """
     class Meta:
-        fields = ('server', 'server_port', 'subnet')
+        fields = ('server', 'server_port', 'server_vpn_ip')
+    subnet = GenericIPNetworkField()
+    # TODO: why is this order not applied?
+    field_order = ('server', 'server_port', 'subnet', 'server_vpn_ip')
```

Comment:
I can't seem to find documentation on this `field_order` option, are you sure this exists?
As an alternative approach (untested) you could try:
```python
class Meta:
   fields = ('server', 'server_port', 'subnet', 'server_vpn_ip')  # i.e. all
   field_classes = {
      'subnet': GenericIPNetworkField,
   }
```
as described here https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#overriding-the-default-fields.

This django admin stuff is great if it just works, but gets rather clunky as soon as you try to do anything non-default with it...